classifier.go

```diff
@@ -75,31 +75,50 @@ type License struct {
 	// Threshold is the lowest confidence percentage acceptable for the
 	// classifier.
 	Threshold float64
+
+	// db is the path to the license database
+	db string
+}
+
+// OptionFunc set options on a License struct.
+type OptionFunc func(l *License) error
+
+// Database is an OptionFunc to specify the location of the license database.
+func Database(db string) OptionFunc {
+	// should we check for db existence here?
+	return func(l *License) error {
+		l.db = db
```

Comment:
Will you ever want to support multiple archive files?

Reply:
Not this week.